docs/src/installation_instructions.md

```diff
@@ -0,0 +1,93 @@
+# Installation
+
+RandomFeatures.jl is a registered Julia package. You can install the latest version
```

Comment:
Not a registered package yet, right? Think only instructions should be to clone the git repo. Unfortunately I lost my version of installation_instructions.md when I rebased #3 .

Reply:
Done